openapi/livestorm/types.bal

```diff
@@ -198,18 +207,69 @@ public type PeopleRequestAttribute record {
     string utm_campaign?;
 };
 
+# 
+public type InlineResponse2003AttributesRegistrantDetail record {
+    # 
```

Comment:
it seems we need to fix the tool, not to generate "#" for blank descriptions in the contract. better to create an issue, but not a priority task IMO

Reply:
I guess removing `description: ''` sections from all the newly added fields should do the trick for now

Reply:
Updated by removing the empty descriptions 17c6380